src/scripts/cfo.zig

```diff
@@ -859,6 +910,11 @@ fn run_fuzzers_start_fuzzer(shell: *Shell, options: struct {
         },
     );
 
+    if (options.fuzzer.capture_logs()) {
+        // Redirect stderr to stdout, so that we only need to read the latter.
+        try std.posix.dup2(process.stderr.?.handle, process.stdout.?.handle);
```

Comment:
Hm, does this work? From reading the man page, I would expect `stdout` to be fully replaced with `stderr`, rather than merged. 

Shouldn't this be happening in the _child_ process instead, here:

https://github.com/ziglang/zig/blob/ee4df4ad3edad160fb737a1935cd86bc2f9cfbbe/lib/std/process/Child.zig#L645-L648

?

This looks very wrong to me, but maybe I misunderstand how POSIX works!